sky/shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -277,22 +331,69 @@ public Binding connectToService(FlutterView view, Core core, MessagePipeHandle p
         });
     }
 
+    /**
+     * Initialize our Flutter config values by searching for a .properties file
+     * containing properties for our configuration variables. If no such
+     * properties file is found in our assets folder, the configuration
+     * variables will retain their default values.
```

Comment:
This will slow down startup in production.  What's the reason to make this configurable?